python/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>MongoDB Charts Embedding Example (Python)</title>
+
+    <script>
+        function renderSignedAuthChart(chartId, iframeId) {
+            let embedUrlRequest = new Request(`/api/embeddedchart/${chartId}`);
+            fetch(embedUrlRequest)
+                .then(response => response.text())
+                .then(url => document.getElementById(iframeId).src = url);
+        }
+    </script>
+</head>
+<body>
+    <h1>MongoDB Charts Embedding Example (Python)</h1>
+    <iframe id="embeddedChart" width="800" height="600" frameborder="0"></iframe>
+
+    <script>
+        renderSignedAuthChart('~REPLACE~CHART_ID', 'embeddedChart'); // Replace first parameter with your Embedded Chart ID
+    </script>
+</body>
```

Comment:
Given these samples are likely to be when people first see the embedding errors, I think we should link to the docs page that explains what these mean. I'll do this across all samples once we have the page to link to.